src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -16,33 +17,61 @@ export interface AdminVenuesProps {
 }
 
 export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
+  const [
+    currentSortingOption,
+    setCurrentSortingOption,
+  ] = useState<VenueSortingOptions>(VenueSortingOptions.az);
+
+  const sortedVenues = useMemo(() => {
+    return sortVenues(venues, currentSortingOption) ?? [];
+  }, [currentSortingOption, venues]);
+
   const renderedPartyVenues = useMemo(
     () =>
-      venues
+      sortedVenues
         ?.filter(isPartyMapVenue)
         .map((venue) => <AdminVenueCard key={venue.id} venue={venue} />),
-    [venues]
+    [sortedVenues]
+  );
+
+  const sortingOptions = useMemo(
+    () => (
+      <DropdownButton variant="secondary" title="Sort venues">
+        {Object.values(VenueSortingOptions).map((sortingOption) => (
+          <Dropdown.Item
+            key={sortingOption}
+            onClick={() => setCurrentSortingOption(sortingOption)}
+          >
+            {sortingOption}
+          </Dropdown.Item>
+        ))}
+      </DropdownButton>
+    ),
+    []
   );
 
   const hasVenues = renderedPartyVenues.length > 0;
 
   return (
-    <div className="admin-venue">
-      <div className="admin-venue__header">
-        <div className="admin-venue__title">Admin Dashboard</div>
+    <div className="AdminVenues">
+      <div className="AdminVenues__header">
+        <div className="AdminVenues__header--left">
```

Comment:
Are there `--left`, `--right`, `--center` modifiers for the header? If not, just a simple name then `&__header-left` or `&__title-wrapper`...

```suggestion
        <div className="AdminVenues__header-left">
```

Reply:
There are no `right` or `center` modifiers but following the BEM pattern `left` should be a modifier, not a part of the element.